src/stack.js

```diff
@@ -1,29 +1,21 @@
 'use strict'
 
 export default class Stack {
-  // your code here
-  // always add and remove to the back of the array
-  // let stack = [];
-  // to push data
-  // stack.push(a)
-  // stack.push(b)
   constructor(){
     this.arr = []
   }
 
-  add(element) {
-    this.arr.push(element)
-  }
+    add(element) {
+      return this.arr.push(element)
```

Comment:
This works, but do you have an idea about how you might do this without using Javascript's `push` method? I think part of the idea of this project is that you think about what push is doing by being explicit and writing it yourself.